README.md

```diff
@@ -1,41 +1,130 @@
   <img src="https://styles.redditmedia.com/t5_2rkse/styles/communityIcon_yq8rp6c884y61.png" width= "100">
   <img src="https://avatars.githubusercontent.com/u/10251060?s=280&v=4" width= "100">
   <img src="https://pbs.twimg.com/profile_images/1326963467119575041/OTgxd3mt_400x400.jpg" width= "100">
   <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTC-B_MIu5Th8IwEU6MZiitQUW5-_sbg_6ldOaldVQViWTjJ5YCWtFQcps_KVVMfBu0H8w&usqp=CAU" width= "100">
-</p>
+</p> -->
+
+## Set up the Microsoft Environment 
 
-## Setup
+1. Install [C# and .NET 8](https://dotnet.microsoft.com/en-us/download)
 
-Install C# and .NET 8 <br>
-https://dotnet.microsoft.com/en-us/download
+2. Install Azure Data Studio && MS SQL Sever [here](https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16)
 
-Azure Data Studio && MS SQL Sever <br>
-https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16
+3. Install [Docker](https://hub.docker.com/)
 
-Docker (Whale Icon at bottom)<br>
-https://www.microsoft.com/en-us/sql-server/sql-server-downloads
 
-<u>Apple Users</u><br>
+### Deployment Tips (If you chose to deploy your own API)
+1.  Before anything create your Microsoft account and set up your [Azure Portal](https://azure.microsoft.com/en-us/get-started/azure-portal)
+  - Sign In or create a Free Account.
+  - A free account comes with $200 USD in credits.
+2.  In your repo have install the [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
+  - Mac
+    - `brew upgrade && brew install azure-cli`
+    - If you're having problems with those commands try running the code one at a time.
+3.  Once you have your portal set up and the azure-cli installed run the following
+  - `az login`
+4.  Once logged in run the following to build your webapp.
+  - `dotnet build --configuration Release`
+5.  Now deploy the app!
+  - `az webapp up --sku F1 --name "YOUR_APP_NAME" --os-type linux`
+    - Replace the OS Type with either windows or linux. For smaller apps there's no difference.
+6. Now go to the Azure Portal. and checkout `App Services`. You should be able to see your new app.
+7. From Azure Portal home page navigate tho ` SQL databases`. Click the plus icon to create your db. Follow the prompts and create your db. 
+  -  Save your password.
+  - Navigate to your db, this will be your new `DefaultConnection` string in your repo.
+
+### Apple Users
 
 If you are working on Mac with a M1 or M2 chip select the 'Apple Chip' options when downloading.
 
-### Guidelines
 
-Since the Database is being hosted with Azure Data Studios you will have to create your own DB! In this repo you will find a fodler called `SQLScripts`, here are the SQL querries that you will need inorder to setup the repo. Copy and Paste them in AzureDataStudios to get setup quickly, or just one at a time. 
+## Database Protocols
```

Comment:
This all looks great. A much more straight forward set up instruction considering how many things you've gotta download and set up.